Sabbath School.xcodeproj/project.pbxproj

```diff
@@ -2627,7 +2655,7 @@
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1002;
 				DEVELOPMENT_TEAM = XVGX5G4YQ9;
-				ENABLE_BITCODE = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
Curious on why did you had to disable bitcode? Is possible to work around it?

Reply:
It has been deprecated with Xcode 14

```Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14.```

https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes

Reply:
It has been deprecated with Xcode 14

```Starting with Xcode 14, bitcode is no longer required for watchOS and tvOS applications, and the App Store no longer accepts bitcode submissions from Xcode 14.
Xcode no longer builds bitcode by default and generates a warning message if a project explicitly enables bitcode: “Building with bitcode is deprecated. Please update your project and/or target settings to disable bitcode.” The capability to build with bitcode will be removed in a future Xcode release. IPAs that contain bitcode will have the bitcode stripped before being submitted to the App Store. Debug symbols for past bitcode submissions remain available for download. (86118779)```

https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes

Reply:
TIL... I guess it's good then 😅